src/main/scala/com/databricks/spark/avro/AvroRelation.scala

```diff
@@ -46,44 +50,6 @@ case class AvroRelation(location: String)(@transient val sqlContext: SQLContext)
     convertedSchema
   }
 
-  /** 
-   * There are several properties of Avro that we have to account for in this method:
-   * 1) Avro uses Utf8 strings, so we want to convert them to java.lang.String for SparkSQL,
```

Comment:
Ah, I guess only the avro map keys thing applies to the moved version of this method. Perhaps just add inline comments for 2 and 3 where appropriate -- I see 1 already has one.